pkg/kubelet/dockershim/helpers.go

```diff
@@ -184,8 +184,8 @@ func makePortsAndBindings(pm []*runtimeapi.PortMapping) (map[dockernat.Port]stru
 // getContainerSecurityOpt gets container security options from container and sandbox config, currently from sandbox
 // annotations.
 // It is an experimental feature and may be promoted to official runtime api in the future.
-func getContainerSecurityOpts(containerName string, sandboxConfig *runtimeapi.PodSandboxConfig, seccompProfileRoot string, separator rune) ([]string, error) {
-	appArmorOpts, err := dockertools.GetAppArmorOpts(sandboxConfig.GetAnnotations(), containerName)
+func getContainerSecurityOpts(containerName string, sandboxConfig *runtimeapi.PodSandboxConfig, seccompProfileRoot, apparmorProfile string, separator rune) ([]string, error) {
```

Comment:
I would drop apparmorProfile from this method, and just add the apparmor opt closer to where appArmor is pulled off the security context. Then this can just be renamed to `getSeccompSecurityOpts`